binder/environment.yml

```diff
@@ -18,5 +18,6 @@ dependencies:
   - glpk
   - gurobi::gurobi
   - pip:
+  - deepcopy
```

Comment:
Isn't this included in Python in any case?
```py
from copy import deepcopy
```

Reply:
will fixt it!